src/DIRAC/ConfigurationSystem/scripts/dirac_admin_check_config_options.py

```diff
@@ -174,6 +174,8 @@ def _printDiff(self, entry, level=""):
             diffType, entryName, _value, changes, _comment = entry
         elif len(entry) == 4:
             diffType, entryName, _value, changes = entry
+        else:
+            raise ValueError(f"Invalid entry {entry}")
```

Comment:
```suggestion
            raise NotImplementedError(f"Invalid entry {entry}")
```

Reply:
The definition of `ValueError` seems more fitting:
> Raised when an operation or function receives an argument that has the right type but an inappropriate value